src/module/hooks/updateHits.ts

```diff
@@ -16,20 +17,21 @@ function getCurrentHits(...args: Record<string, any>[]) {
 }
 
 Hooks.on('preUpdateActor', async (actor:TwodsixActor, update:Record<string, any>) => {
-  if (update.data?.characteristics) {
-    update.data.hits = getCurrentHits(actor.data.data.characteristics, update.data.characteristics);
+  if (update.data?.characteristics && actor.type=== 'traveller') {
```

Comment:
As a thought experiment, I've been thinking about whether we need to have NPC, robot and animal actors.  Do we need to check if a traveller? Or maybe better !="ship" ?